queries.sql

```diff
@@ -1,3 +1,6 @@
-/*Queries that provide answers to the questions from all projects.*/
+/* Populate database with sample data. */
 
-SELECT * from animals WHERE name = 'Luna';
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', 'February 3, 2020', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', 'November 15, 2018', 2, true, 8);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 'January 7, 2021', 1, false, 15.04);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, true, 11);
```

Comment:
- [ ] Great job so far testing the queries and adding the screenshots in your PR but I can't see here that you wrote any queries so I think it would be great if you write it here in query.sql file all the solved questions  💯✨✨🔥👍🏻♻